src/data/github-projects.json

```diff
@@ -193,6 +193,25 @@
             "repoURL": "https://github.com/badges/shields",
             "logo": "sheilds-io.svg"
         },
+        {
+            "title": "A-to-Z-Resources-for-Students",
+            "description": "Curated list of resources for college students.",
+            "tags": [
```

Comment:
I think this project also has an `awesome` badge. Maybe it should be included here?